docs/api-security/14-FHIRSecurity.md

```diff
@@ -71,8 +71,29 @@ FHIR provides an [AuditEvent](https://build.fhir.org/auditevent.html) resource s
 
 ## Access control considerations
 
+### SMART on FHIR Scopes
+
+HL7 International produce a standard for [App Launch called SMART](https://hl7.org/fhir/smart-app-launch/). SMART uses [scopes to control access to resources](https://hl7.org/fhir/smart-app-launch/scopes-and-launch-context.html). <ApiStandard id="HNZAS_SHOULD_USE_SMART_ON_FHIR" type="SHOULD" toolTip="Where API Providers are delivering FHIR APIs they SHOULD consider the use of SMART scopes.">Where API Providers are delivering FHIR APIs they **SHOULD** consider the use of SMART scopes.</ApiStandard>
+
+Examples of SMART on FHIR Scopes:
+- `patient/Patient.r` : A patient viewing their own `Patient` resource
+- `system/Observation.c` : A system creating `Observation` resources 
+
+#### SMART Scopes for Operations
+While SMART on FHIR provides a robust set of rules for CRUDS operations, the standard is still emerging as how best to represent Operations using the same syntax.<ApiStandard id="HNZAS_MUST_USE_OPDEF" type="MUST" toolTip="Operations MUST be authorised with the OperationDefinition URL as the scope string.">Where API Providers are authorising FHIR Operations with SMART on FHIR, they **MUST** use the URL of the OperationDefinition as the scope string.</ApiStandard>
```

Comment:
```suggestion
While SMART on FHIR provides a robust set of rules for CRUDS operations, the standard is still emerging as how best to represent Operations using the same syntax.<ApiStandard id="HNZAS_MUST_USE_OPDEF" type="MUST" toolTip="Operations MUST be authorised with the OperationDefinition canonical URL as the scope string.">Where API Providers are authorising FHIR Operations with SMART on FHIR, they **MUST** use the canonical URL of the OperationDefinition as the scope string.</ApiStandard>
```

Reply:
Should we add an example? Maybe `https://implementation-guides.digital.health.nz/fhir-nzps/OperationDefinition/patient-summary`